processor/k8sattributesprocessor/internal/kube/client.go

```diff
@@ -222,21 +223,11 @@ func (c *WatchClient) Start() error {
 			return err
 		}
 		synced = append(synced, reg.HasSynced)
-		go c.replicasetInformer.Run(c.stopCh)
+		waitForInformers = append(waitForInformers, c.replicasetInformer)
+		go c.runInformer(c.replicasetInformer)
```

Comment:
Do we need the extra function call here? Maybe we can just call directly the `go c.replicasetInformer.Run(c.stopCh)`?